src/main/java/org/opensearch/security/dlic/rest/api/SecurityConfigAction.java

```diff
@@ -69,7 +71,9 @@ public SecurityConfigAction(final Settings settings, final Path configPath, fina
 
     @Override
     public List<Route> routes() {
-        return allowPutOrPatch ? allRoutes : getRoutes;
```

Comment:
Please use similar approach as for all other `routes()`: routes should return statically defined list.